src/grammar.lua

```diff
@@ -453,3 +456,17 @@ function pmatch(patt)
     end
   end)
 end
+
+_M.anyOf	= anyOf
```

Comment:
1) why `_M` ?

I use the generic `_M` for the module just because I convert lot of modules. It's easier make the same approach for every modules. Easier copy/paster of code if needed. I never need to check the name of the current module to check the module exporting stuff.
If the module code already use a kind of table name, I play with :
```lua
local _M = {}
local grammar = _M

-- the code use the gammar name like gammar.anyOf ...

return _M
```

or the opposite :
```lua
-- existing code with local grammar = {} somewhere
local _M = grammar

-- existing code that use grammar.anyOf

return _M
```

It should be a more sexy name like `public` or `export` ... I don't know, I just use `_M` everywhere :)

2) why my approach is fast to convert ?

I do my patch with lot of `local var1, var2, func1, func2, ...` because if you use `function _M.anyOf()` instead of `local anyOf` at the beginning and `_M.anyOf = anyOf` at the end,
you must find and replace all `anyOf` call by `_M.anyOf`.
When you have a lot of number of functions/variables to localize it was really boring and spend more time.

3) my way is bad in one case
The only case that is mandatory to use _M.anyOf instead of localized anyOf value is when you need to allow overwritting of the function value by a external module/code.

Sample of code :
```lua
local gammar = require "lep.grammar"
local function myfunc(...)
   -- your code
end
grammar.anyOf = myfunc
```
My approach ignore the myfunc value in the internal grammar work.
I usual case this overwriting case is not wanted.

I think this case with my current approach should be fixed by using a specific `__newindex` metatable for the module to apply overwriting value to the localized variable.

4) my patch should be improved
... but it require time and it should be done later !